src/test/java/pages/ManageFiltersPages.java

```diff
@@ -15,11 +16,19 @@
 
     public void clickbuttonDeleteApprove(){ $(By.cssSelector("#delete-filter-submit")).click(); }
 
-    public void checkAvailabilityFilter(){
-        $(By.linkText("1 testSaveFilter")).shouldBe(Condition.visible); }
+    public void checkAvailabilityFilter(String request){
```

Comment:
Maybe better to use method overloading instead of rewriting it? Do other tests works fine?

Reply:
This method was created specially for my test and was used only for one filter "1 testSaveFilter". After rewrote mentioned method became more universal.